addons/event/report/event_event_templates.xml

```diff
@@ -26,6 +26,8 @@
                                             <h3 t-field="o.name"/>
                                         </small>
                                     </div>
+                                     <div class="col-xs-12 text-center" style="background: lightgrey;" id="o_event_barcode">
```

Comment:
Why adding void div here, it will add lightgrey line after event name unnecessarily, I think the goal of this div is to add barcode image so why not add whole div with barcode image using xpath after o_event_name(wherever you are adding image)
``` <xpath expr="//div[@id='o_event_name']" position="after">
         ...
     </xpath>
```